scripts/build/companion_libs/050-zlib.sh

```diff
@@ -84,6 +84,7 @@ do_zlib_backend() {
     CT_DoLog EXTRA "Configuring zlib"
 
     CT_DoExecLog CFG                                  \
+    CC="${host}-gcc"                                  \
```

Comment:
I'll have a look, but I'd point out that we build canadian crosses successfully where using "gcc" would have resulted in a library for a different architecture - that would fail during the link.